src/vsg/platform/win32/Win32_Window.cpp

```diff
@@ -149,32 +149,32 @@ KeyboardMap::KeyboardMap()
             {VK_RIGHT                             ,              KEY_Right    },
             {VK_DOWN                              ,              KEY_Down     },
             {VK_SELECT                            ,              KEY_Select},
-            {VK_PRINT                             ,              KEY_Print},
+            {VK_SNAPSHOT                          ,              KEY_Print},
             {VK_EXECUTE                           ,              KEY_Execute},
             {VK_SNAPSHOT                          ,              KEY_Undefined},
             {VK_INSERT                            ,              KEY_Insert},
             {VK_DELETE                            ,              KEY_Delete},
             {VK_HELP                              ,              KEY_Help},
             {VK_LWIN                              ,              KEY_Super_L},
             {VK_RWIN                              ,              KEY_Super_R},
-            {VK_APPS                              ,              KEY_Undefined},
+            {VK_APPS                              ,              KEY_Menu},
```

Comment:
Um... VK_Menu isn't a menu but VK_APPS is?

Reply:
Please see [this comment above](https://github.com/vsg-dev/VulkanSceneGraph/pull/1248#discussion_r1694315657)